apps/ledger-live-mobile/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # live-mobile
 
+## 3.5.0-hotfix.0
```

Comment:
Weird we have 3.5.0-hotfix.0 in the changelog file and 3.4.1-hotfix.0 in the package.json